client/components/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@automattic/simple-components",
+	"version": "0.1.0-alpha.5",
+	"description": "React components, as used on WordPress.com",
+	"scripts": {
+		"build": "npx babel --only ./index.js,./button/,./main/ --out-dir cjs .",
```

Comment:
with the [babel 7 beta 53 upgrade](https://github.com/Automattic/wp-desktop/pull/477), this is going to get more complicated. babel will now pull the config from the root of the repo, in `babel.config.js` instead of looking for the closest `.babelrc.js`. So far we don't have a `.babelrc.js`, but we may want one.

How does this package handle polyfills? We're generating them at the entry point in calypso, but for a package like this, we probably want them polyfill'd at the usage point? Or not at all and we require the client to polyfill what we need?

Reply:
> How does this package handle polyfills?

Badly :laughing: Just been struggling with the following error

```
ERROR in ./client/boot/polyfills.js
Module not found: Error: Can't resolve 'core-js/modules/es6.array.sort' in '~/src/wp-calypso/client/boot'
```

So yeah, generating at entry point is probably the least desirable...

> Or not at all and we require the client to polyfill what we need?

That seems kinda reasonable...

Reply:
Just filed #26094 which should fix this.